docs/intro/contributing.txt

```diff
@@ -26,9 +26,7 @@ Who's this tutorial for?
 For this tutorial, we expect that you have at least a basic understanding of
 how Django works. This means you should be comfortable going through the
 existing tutorials on :doc:`writing your first Django app</intro/tutorial01>`.
-In addition, you should have a good understanding of Python itself. But if you
-don't, `Dive Into Python`__ is a fantastic (and free) online book for
-beginning Python programmers.
+In addition, you should have a good understanding of Python itself.
```

Comment:
Should this `Python` string be a link to python's main site?

Reply:
In this context, I guess no. The reason being, the snippet mainly focuses on Django, so Python is just a reference word.